libs/util/project.json

```diff
@@ -23,12 +23,12 @@
     },
     "test": {
       "executor": "@nrwl/jest:jest",
-      "outputs": ["coverage/libs/util"],
+      "outputs": ["coverage"],
```

Comment:
Why is it like this here? I noticed that constants have a different path

Reply:
These output properties in project.json file are removed. coverage directories are already defined in jest.config.ts
fixed de1f40717ab164f66408e3bf2acbe282ff4ce7e7

Reply:
Wrong commit ref. Points to [style(data models/sockets): update naming convention of types](https://github.com/the-ai-team/razor/commit/de1f40717ab164f66408e3bf2acbe282ff4ce7e7)

Reply:
refactor: remove unnecessary path defined for coverage [ecf12984f9c184cb98b3a1f916ff938d94279cc9]